app/client/cli/gov.go

```diff
@@ -0,0 +1,69 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/pokt-network/pocket/utility/types"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+)
+
+func init() {
+	rootCmd.AddCommand(NewGovernanceCommand())
+}
+
+func NewGovernanceCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Governance",
+		Short:   "Governance specific commands",
+		Aliases: []string{"gov"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(govCommands()...)
+
+	return cmd
+}
+
+func govCommands() (cmds []*cobra.Command) {
+
+	cmds = append(cmds, &cobra.Command{
+		Use:   "ChangeParameter <owner> <key> <value>",
+		Short: "ChangeParameter <owner> <key> <value>",
+		// DISCUSS(deblasis): do we need some sort of validation on the backend?
+		Long:    "Changes the Governance parameter with <key> owned by <owner> to <value>",
+		Aliases: []string{},
+		Args:    cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(deblasis): implement RPC client, route and handler
+			fmt.Printf("changing parameter %s owned by %s to %s\n", args[1], args[0], args[2])
+
+			// TODO(deblasis): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+
+			key := args[1]
+			value := args[2]
+
+			pbValue, err := anypb.New(wrapperspb.String(value))
+			if err != nil {
+				return err
+			}
+
+			_ = &types.MessageChangeParameter{
+				Signer:         []byte{}, // TODO(deblasis): 👀 same as owner?
```

Comment:
Can you add a `TODO(andrew)` in `message.proto` to document the difference? 

For now, you can use the same ones.

The `owner` is the owner of the funds staked (where the rewards go) and the `signer` is the operator. Most of the time, if you run your own node, it's the same thing.

Reply:
I don't think it's an AI for Andrew, your explanation confirms my hunch and it makes total sense.

These "TODO 👀" are a reminder for me to dig deeper before raising questions (for which comments like yours make my life easier tho). Normally I wouldn't share but since I felt this PR could be a bit of a blocker, I wanted to share as much as possible so that you guys can figure out where we are at with this, including my doubts hehe.

One thing I figured is that I think I have to construct these messages server side and not on the client for various reasons:
- implementation details leaks + tight coupling
- on the server side (node) we know the signer address so I can populate this value correctly I guess